core/domain/user_services.py

```diff
@@ -1060,6 +1060,11 @@ def update_profile_picture_data_url(user_id, profile_picture_data_url):
         user_id: str. The unique ID of the user.
         profile_picture_data_url: str. New profile picture url to be set.
     """
+
+    if profile_picture_data_url == '':
+        user_email = get_email_from_user_id(user_id)
+        profile_picture_data_url = fetch_gravatar(user_email)
```

Comment:
Hmm instead of fetching the gravatar here I would use `DEFAULT_IDENTICON_DATA_URL` here.

Reply:
Also, backend tests should be added. And maybe instead of depending on `profile_picture_data_url == ''` here, create a separate function for this.

Reply:
Done! Now, I have used `DEFAULT_IDENTICON_DATA_URL` and also added a corresponding backend test.
As we set `profile_picture_data_url` to `''` in the frontend, we have to depend on `profile_picture_data_url == ''` condition.

Reply:
But you can add a separate handler for removing the profile picture, right?

Reply:
Hey @vojtechjelinek, as we already have get and set handlers for the profile picture data url, then why do we need to add a new one as this functionality is only to change the profile picture data url to DEFAULT_IDENTICON_DATA_URL? Also, if we add a backend handler then we have to change almost all the frontend files calling profile data api endpoints and backend files related to preference page, profile picture and user services.

Reply:
Hmm, I think it would make more sense to actually implement this as a separate handler, since in the future we want to refactor how our handling of pictures works.

Reply:
But I think that it is fine to merge this as is.